src/client/training.py

```diff
@@ -11,54 +11,66 @@
+    parameter.
+
+    For more information, see [Adding a training](https://toloka.ai/en/docs/guide/concepts/train).
 
     Attributes:
-        project_id: ID of the project to which the training pool belongs.
-        private_name: Training pool name (only visible to the requester).
-        may_contain_adult_content: The presence of adult content in learning tasks.
-        assignment_max_duration_seconds: Time to complete a set of tasks in seconds.
-            It is recommended to allocate at least 60 seconds for a set of tasks
-            (taking into account the time for loading the page, sending responses).
-        mix_tasks_in_creation_order: The order in which tasks are included in sets:
-            * True - Default behavior. Include tasks in sets in the order they were loaded.
-            * False - Include tasks in sets in random order.
-        shuffle_tasks_in_task_suite: Order of tasks within the task set:
-            * true - Random. Default behavior.
-            * false - The order in which the tasks were loaded.
-        training_tasks_in_task_suite_count: The number of tasks in the set.
-        task_suites_required_to_pass: The number of task suites that must be successfully completed to assign a skill
-            and access the main tasks.
-        retry_training_after_days: After how many days the replay will become available.
-        inherited_instructions: Indicates whether to use a project statement.
-            If training need their own instruction, then specify it in public_instructions. Default value - False.
-        public_instructions: Instructions for completing training tasks. May contain HTML markup.
-        metadata:
-        owner: Training pool owner.
-        id: Internal ID of the training pool. Read only.
-        status: Training pool status. Read only.
-        last_close_reason: The reason the training pool was last closed.
-        created: UTC date and time of creation of the training pool in ISO 8601 format. Read only.
-        last_started: UTC date and time of the last start of the training pool in ISO 8601 format. Read only.
-        last_stopped: UTC date and time of the last stop of the training pool in ISO 8601 format. Read only.
+        project_id: The ID of the project containing the training.
+        private_name: The training pool name. It is visible to the requester only.
+        may_contain_adult_content: The presence of adult content.
+        assignment_max_duration_seconds: Time limit to complete a task suite.
+            Take into account loading a page with a task suite and sending responses to the server. It is recommended that you set at least 60 seconds.
+        mix_tasks_in_creation_order:
+            * True — Tasks are grouped in suites in the order they were created.
+            * False — Tasks are chosen for a task suite in a random order.
+
+            Default: False.
```

Comment:
As you suggested earlier the default value is `True` (the default is on the backend side, client-side the default is `None`, which means the field is not added to the request body)